packages/big-design/src/components/Table/Actions/Actions.tsx

```diff
@@ -58,17 +59,19 @@ const InternalActions = <T extends TableItem>({
     <StyledFlex
       alignItems="center"
       aria-controls={tableId}
+      aria-label="Table Controls"
```

Comment:
🍹 We should localize this before we release, but you can tackle that in another PR if you want.

Reply:
I think a seperate PR would be best, as we'll need to update the props of all `Table` variants in one go